doc/devel/index.rst

```diff
@@ -16,13 +16,26 @@ Contribute
 :octicon:`heart;1em;sd-text-info` Thank you for your interest in helping to improve
 Matplotlib! :octicon:`heart;1em;sd-text-info`
 
-There are various ways to contribute: optimizing and refactoring code, detailing
-unclear documentation and writing new examples, helping the community, reporting
-and fixing bugs and requesting and implementing new features...
+This project is a community effort, and everyone is welcome to contribute. Everyone
+within the community is expected to abide by our :ref:`code of conduct <code_of_conduct>`.
+
+There are various ways to contribute, such as optimizing and refactoring code,
+detailing unclear documentation and writing new examples, helping the community,
+reporting and fixing bugs, requesting and implementing new features...
 
 .. _submitting-a-bug-report:
 .. _request-a-new-feature:
 
+Issue tracker
```

Comment:
Totally optional, but GitHub is so well known, that adding the name may be more obvious to people that just an "abstract" concept of issue tracker.

```suggestion
GitHub issue tracker
```